quicksort/quicksort.c

```diff
@@ -15,19 +15,43 @@
     
     Do not just use the `qsort` function from the standard library.
```

Comment:
Even though it said not to, I think you should go and do it again and use the standard library.. It's always a wise idea to know what each language has in it's tool kit.